cartridges/int_adyen_overlay/cartridge/scripts/adyenGetConnectedTerminals.ds

```diff
@@ -0,0 +1,74 @@
+/**
+* Send request to adyen to get connected terminals based on merchant account and storeId
+*
+*/
+
+//script include
+var AdyenHelper = require("*/cartridge/scripts/util/AdyenHelper");
+var Logger = require("dw/system/Logger");
+
+function getTerminals() {
+    try {
+        var requestObject = {};
+
+        var callResult = null,
+            resultObject = null,
+            service = AdyenHelper.getService(AdyenHelper.SERVICE.CONNECTEDTERMINALS);
+
+        if (service == null) {
+            Logger.getLogger("Adyen").error("Could not retrieve connected terminals, error creating the service");
+            return;
+        }
+        //set merchant account & optionally storeId
+        requestObject["merchantAccount"] = AdyenHelper.getAdyenMerchantAccount();
+
+        if(AdyenHelper.getAdyenStoreId() != null){
+            requestObject["store"] = AdyenHelper.getAdyenStoreId()
+        }
+
+        var xapikey = AdyenHelper.getAdyenApiKey();
+        service.addHeader("Content-type", "application/json");
+        service.addHeader("charset", "UTF-8");
+        service.addHeader("X-API-key", xapikey);
+        callResult = service.call(JSON.stringify(requestObject));
+
+        if (callResult.isOk() == false) {
+            Logger.getLogger("Adyen").error("Adyen: Call error code" +  callResult.getError().toString() + " Error => ResponseStatus: " + callResult.getStatus()  + " | ResponseErrorText: " +  callResult.getErrorMessage() + " | ResponseText: " + callResult.getMsg());
+            return ;
```

Comment:
This could throw an error instead, so that you would have a centralized logging code.